src/db/repositories/base-methods.repository.js

```diff
@@ -11,11 +11,43 @@ function baseMethods (tableName, database) {
     }
     return data.rows
   }
+  const create = async (opt) => {
+    let data = null
+    let values = Object.keys(opt).reduce((acc, cur, index) => acc + `$${index + 1},` , "").slice(0,-1);
```

Comment:
copy/paste
the solution for avoid copy past:
```suggestion
    const createValuesSql = (opt, cbTemplate) => Object.keys(opt).reduce((acc, cur, index) => acc + cbTemplate(acc, cur, index), "" ).slice(0, -1)
```
example of using:
`const values = createValuesSql(opt, (acc, cur, index) => '$${index + 1}')`